webapp/src/shared/components/Input/EditUserForm.tsx

```diff
@@ -34,7 +38,7 @@ export default function EditUserForm({
     let txt;
     if (
       window.confirm(
-        `Your name: "${fullName}" and your email: "${email}".\n
+        `Your Username: "${username}", Full name: "${fullName}", and your email: "${email}".\n
```

Comment:
When we try to change our username to match an already existing username, the update will not work, as it should, but the alert will still say "Your data has been saved".
Maybe we should should make some warning message when the username the user tries to set already exists in the database?

Reply:
Thank you, I'll look into it.

One option is to check if the API call succeeded and set a valid status message or an invalid one.

For example:
Update successfully
The provided info is invalid, missing, or already exists.

If we want more detailed messages, we could check the error.response.status:
400 Bad Request -> Invalid or missing info
422 Unprocessable Entity -> Info already exists

And if we want even more detailed messages, we have to edit the response object and parse the database error message.

Reply:
> 400 Bad Request -> Invalid or missing info
422 Unprocessable Entity

In my opinion this should be enough. Unprocessble entity could be an array of different bad input, which is not very specific (could be duplicated username, or malicious attempt at SQL injection detected, or non displayable characters, or too long data), but I don't think it's crucial to display complete error information here. Maybe I'm wrong.

Reply:
I made these changes. Please let me know what we think about this solution.